src/lexer.jl

```diff
@@ -1021,39 +1004,42 @@ function lex_cmd(l::Lexer, doemit=true)
     end
 end
 
+const MAX_KW_LENGTH = 10
 function lex_identifier(l::Lexer{IO_t,T}, c) where {IO_t,T}
     if T == Token
         readon(l)
     end
```

Comment:
It's not strictly part of this PR but I just noticed the `T == Token` check here.  Why do we have this here when `readon(l)` already contains the logic for `Token` vs `RawToken`?